rust-bindings/src/lib.rs

```diff
@@ -0,0 +1,172 @@
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+#![allow(dead_code)]
+#![allow(non_upper_case_globals)]
+
+extern crate link_cplusplus;
+
+mod bindings {
+    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+}
+
+pub fn validate_proof(
+    seed: &[u8; 32],
```

Comment:
In all the code and literature this is referred to as "plot_id" or "id", it think the same nomenclature should be retained here.